stable/admin/templates/statefulset.yaml

```diff
@@ -198,3 +209,28 @@ spec:
       resources:
         requests:
           storage: {{ .Values.admin.persistence.size }}
+{{- if .Values.admin.logPersistence.enabled }}
+  - metadata:
+      name: log-volume
```

Comment:
How many volumes is this creating per admin deployment? I see the log-volume mounted also in the init container. Is it reused?

Reply:
That logic is unchanged, I guess it is just re-used between them...

Reply:
Every container in a pod sees the same disks.

One common usage of an initContainer is to precondition the volumes that will be used by the main container.

The SM initContainer ensures that the archive mount has the correct permissions, for example.